packages/lib-user/src/components/shared/Layout/components/PageHeader.js

```diff
@@ -0,0 +1,56 @@
+import { Box, ResponsiveContext } from 'grommet'
+import { useContext } from 'react'
+import { arrayOf, node } from 'prop-types'
+import { ZooniverseLogotype } from '@zooniverse/react-components'
+import styled from 'styled-components'
+
+import HeaderDropdown from './HeaderDropdown'
+
+const HeaderItems = styled(Box)`
+  max-width: calc(90rem - 160px);
+`
+
+function PageHeader({ primaryHeaderItem = '', secondaryHeaderItems = [] }) {
+  const size = useContext(ResponsiveContext)
+
+  return (
+    <Box
+      as='header'
+      background='neutral-1'
+      fill
+      height={{ min: '140px', max: '140px' }}
+      align='center'
+      pad={{ horizontal: 'medium' }} // Matches ZooHeader and body horizontal padding
+    >
+      <HeaderItems
+        direction='row'
+        fill
+        height={{ min: '70px', max: '70px' }}
+        align='center'
+        justify='between'
+      >
+        <Box align='center' direction='row'>
+          {primaryHeaderItem}
+        </Box>
+        <Box align='center' justify='end' direction='row' gap='small'>
+          {secondaryHeaderItems.length > 0 ? (
+            size !== 'small' ? (
+              secondaryHeaderItems.map(HeaderItem => HeaderItem)
+            ) : (
```

Comment:
```suggestion
            size !== 'small' ? (secondaryHeaderItems) : (
```
I unnecessarily included the `map` in earlier work; since it's an array of nodes, I think the array can be returned without the `map`. Not a necessary change, for consideration.

Reply:
Will do!